bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueBindingConstants.java

```diff
@@ -107,4 +128,46 @@ public class HueBindingConstants {
 
     // Config status messages
     public static final String IP_ADDRESS_MISSING = "missing-ip-address-configuration";
+
+    // thing types for CLIP 2
+    public static final ThingTypeUID THING_TYPE_CLIP2 = new ThingTypeUID(BINDING_ID, "clip2");
+    public static final ThingTypeUID THING_TYPE_DEVICE = new ThingTypeUID(BINDING_ID, "device");
+    public static final ThingTypeUID THING_TYPE_GROUPED_LIGHT = new ThingTypeUID(BINDING_ID, "groupedLight");
+
+    // channels for CLIP 2
+    public static final String CHANNEL_2_BATTERY_LEVEL = "batterylevel";
+    public static final String CHANNEL_2_BATTERY_LOW = "batterylow";
+    public static final String CHANNEL_2_BUTTON_LAST_EVENT = "buttonLastEvent";
+    public static final String CHANNEL_2_COLOR_TEMPERATURE = "colorTemperature";
+    public static final String CHANNEL_2_COLOR_TEMPERATURE_ABS = "colorTemperatureAbs";
+    public static final String CHANNEL_2_LAST_UPDATED = "lastUpdated";
+    public static final String CHANNEL_2_LIGHT_LEVEL = "lightLevel";
+    public static final String CHANNEL_2_LIGHT_LEVEL_ENABLED = "lightLevelEnabled";
+    public static final String CHANNEL_2_MOTION = "motion";
+    public static final String CHANNEL_2_MOTION_ENABLED = "motionEnabled";
+    public static final String CHANNEL_2_ROTARY_STEPS = "rotarySteps";
+    public static final String CHANNEL_2_TEMPERATURE_ENABLED = "temperatureEnabled";
+    public static final String CHANNEL_2_ZIGBEE_STATUS = "zigbeeStatus";
+
+    public static final String CLIP2_THING_TYPE_PREFIX = "hue:clip2:";
+    public static final String CLIP2_PROPERTY_SUFFIX = " (api2)";
```

Comment:
It is there because of the discovery service. Bridges are discovered via mDNS, and once a bridge has been discovered, it checks if the bridge actually supports API v2. If so, an API v2 bridge is put in the inbox (marked with the 'apiv2' suffix). but otherwise it puts an API v1 bridge in the inbox without a suffix.

However, I am not dogmatic about this, and can happily remove it if you think it makes sense.

Reply:
removed